.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "search.exclude": {
+        "coverage/": true
+    },
+}
```

Comment:
@orta This file you should [ignore globally](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) in your Git setup 💡 .